src/main/java/api/apiwrappers/OrderApiWrapper.java

```diff
@@ -0,0 +1,21 @@
+package api.apiwrappers;
+
+import api.CommonApiWrapper;
+import api.domain.store.Order;
+
+import java.io.IOException;
+
+public class OrderApiWrapper extends CommonApiWrapper {
+
+    public Object postOrder(Order order) throws IOException {
+        String endpoint = "https://petstore.swagger.io/v2/store/order";
```

Comment:
un endpoint completeaza un baseUrl unic pentru tot proiectul (`baseUrl=https://petstore.swagger.io` il putem pune in properties file)